docs/adr/0016-publishing-times-and-change-history.md

```diff
@@ -0,0 +1,208 @@
+each of these times. A consequence of these not matching is that migrated
+content would show a different first publishing time.
+
+#### 2. Migrating change notes from Whitehall
+
+Whitehall doesn't make use of the [Publishing API ChangeNote][pub-api-change-note]
+model to build a change history, instead Whitehall creates it's own change
+history for content and includes it when updating content with the Publishing
+API. This has a consequence that the data stored in ChangeNote models in
+Publishing API for Whitehall content cannot be relied upon as there are no
+guarantees of its accuracy. There is also not an expectation that this data is
+accurate as many pieces of Whitehall content were created before the
+ChangeNote model was introduced to Publishing API.
+
+For Content Publisher to migrate change histories for Whitehall content it
+would need to ensure the entire change history is in sync with the Publishing
+API.
+
+#### 3. Removing/editing change notes
+
+It is a relatively common task that GOV.UK 2nd line support engineers are
+[requested to remove or modify a change note][change-note-docs] for a published
+piece of content. To resolve this in Whitehall a developer would modify a past
+edition of a piece of content to either remove or amend the change note.
+
+In Content Publisher it is not ideal to modify a past edition. An edition
+represents a snapshot of the content as it was published on GOV.UK and any edits
+invalidate the accuracy of this snapshot.
+
+#### 4. Backdating content
+
+The backdating feature in Content Publisher supports only backdating the first
+edition of a piece of content and not subsequent ones as is allowed by
+Whitehall. This is because the Publishing API does not have the means to change
+the initial item in the change history for content and thus would cause a
+change history that is inconsistent with the `first_published_at` time. This
+has a consequence for publishers that if they make a mistake with backdating
+or forget to backdate in Content Publisher then they cannot rectify this
+without creating a new document or requesting support from developers.
+
+### Approaches considered
+
+#### 1. Correct Whitehall data in the Publishing API
+
+We considered doing an audit and syncing exercise for times and change
+histories from Whitehall to the Publishing API. This would involve either
+doing a large data modification exercise or creating new endpoints
+in the Publishing API that allow importing this data.
+
+The positive of this approach was that it could mean Content Publisher
+could continue to rely on the Publishing API for time and change history
+management without needing this data stored in Content Publisher.
+
+The negatives to this were that there would be no guarantees that this data
+would remain accurate once the work was complete, as Whitehall continues to
+operate; nor would there be an easy way to check whether times are still
```

Comment:
Minor: I'm not really sure what we mean by 'easy', since we seem to be calling the API for data integrity checks already. The latter argument sways it for me anyway, so I'm not worried if it's too complicated to explain here.

Reply:
I've made an amend. 

The reason it's not easy is that the Content Publisher edition wouldn't have the data stored so to integrity check it you'd need to compare the data Whitehall has against the Publishing API. We could of course access that from our Whitehall dump of data but that would also be complex as that is not the subject in use at the point of integrity checking.